src/ln/msgs.rs

```diff
@@ -585,6 +586,9 @@ pub trait RoutingMessageHandler : Send + Sync {
 	fn handle_channel_update(&self, msg: &ChannelUpdate) -> Result<bool, HandleError>;
 	/// Handle some updates to the route graph that we learned due to an outbound failed payment.
 	fn handle_htlc_fail_channel_update(&self, update: &HTLCFailChannelUpdate);
+	///This returns a vec tuple of (ChannelAnnouncements,  ChannelUpdates, NodeAnnouncements NodeAnnouncements). It also returns the id of the last channel passed through
```

Comment:
"Gets a subset of the messages required to dump our routing table to a remote node, starting at the short_channel_id indicated by starting_point and including batch_amount entries"? Or something else that explains what you have to do here, not just the return value (which is written in the code, so probably doesn't need redundant documentation to understand...).